### DISCUSSION TITLE
[CHANGE] Accessible and Comprehensive FAQ Section

### DISCUSSION THREAD

**Discussion by hujikol** [sunflower-land/sunflower-land#4023] [Ideas]
## How the Feature is Currently Working

- For now, we have access to get to the official docs from the game only from the "How to play the game" and even it stays on the last page.
     > it got 5 steps to get to the docs or 4 steps to get the doc links
     <img src="https://user-images.githubusercontent.com/12151051/182079909-cf3e026a-a2ea-4904-aeaa-42e4d814f08b.png" width="360">
- There is no button to directly refer to the FAQ section.
- The docs got a [FAQ section](https://docs.sunflower-land.com/support/faq) but it only shows an introduction to the game and not the actual FAQ itself, but also we got another FAQ section but on a separate page, it is [Freshdesk page](https://sunflowerland.freshdesk.com/support/solutions/101000195014).

## How the Feature Should Be Changed

- Adding an accessible button in the game menu to get to the mega docs guide
- Make the sum up of all FAQs based on the [Freshdesk page](https://sunflowerland.freshdesk.com/support/solutions/101000195014) and discord channel [#FAQ](https://discord.com/channels/880987707214544966/964529940538228766).
- Update the docs every new release or changes
- Searchable FAQ topics (Freshdesk basic function)

## Proof of Community Demand

I don't have any pools regarding this change, but from my point of view, it is good to have a more structured and simplified FAQ section about the game itself. The discord channel is still needed to assist the unconditional bugs and errors that are not frequent or expected to happen.

it is so many similar questions asked in the discord channel while the discord has a FAQ channel but it is a bit too overwhelming for the farmers

if such nice and clear documentation can be built, it will bring no confusion to our farmers and also limit the frequent problem report on the discord channel, by the time we can solve problems by only referring to the FAQ section.

## Technical Requirements

1. Adding a new section in the PR to get the docs updated by the chore team or someone who has the access to it.

```
## Changes to the documentation
> Removes this section if not needed

- TODO
```

## UI/UX Design Requirements

1. Adding the "Help" button on the in-game menu.
<img src="https://user-images.githubusercontent.com/12151051/182080231-4368a6f7-6919-410c-b492-d20891982e40.png" width="240">

**Comment by adamhannigan** [comment]
@hujikol Good suggestion.

We aim to keep the menu's lightweight and don't want to too many buttons. In this case we would choose between either "How to play" or a "Help" button to link to the docs. Which do you think is better?

Also, do you have any suggestions how the docs themselves can be improved?

**Comment by hujikol** [comment]
Thank you for following up on this issue sir @adamhannigan , 
I think it is better to omit the "How to play" button then warp up on the "Help" button to link the docs, then it will link to see the "How to play" section while at the end of it we can also refer into the FAQ section

something like this
<img src="https://user-images.githubusercontent.com/12151051/183122439-4680296f-c971-4124-b3f7-02c134db0020.png" width="480">

For the docs, I think it's just alright to be like that but for specific the F.A.Q. section, I think it needs some revamp, to be more compact and not have separate multiple websites, 

like I stated in this issue, the docs got a [FAQ section](https://docs.sunflower-land.com/support/faq) on the docs but it only shows an introduction to the game and not the actual FAQ itself, but also we got another FAQ section but on a separate page, it is the [Freshdesk page FAQ](https://sunflowerland.freshdesk.com/support/solutions/101000195014).

Can we just sum up the FAQ from Freshdesk into the docs.sunflower-land/faqs ?, or the other way, from the docs into the Freshdesk, so FAQ button in the docs.sunflower-land will be referring to the Freshdesk website...

**Comment by adamhannigan** [comment]
@hujikol Apologies, I missed the response.

Yes I agree on this. We can link to the docs and default them to a "How to play" section and reference the FAQ.

We definitely need some better docs around the FAQ. I will chat with the team 😄

**Comment by hujikol** [comment]
thankyou for the positive feedback sir @adamhannigan, really appreciates the hard work of the team!

hopefully, I can do some favor too someday 😄

**Comment by stevew00dy** [comment]
![image](https://user-images.githubusercontent.com/5225886/202224426-0c432ce7-169e-40ed-94eb-a30df89bf6e1.png)

I would remove the icon - it's the only menu to have one and the right align looks out of place.

**Comment by stevew00dy** [comment]
Here are my recommendations for this section

How to Farm

1, Harvest crops when they are ready
2, Visit the shop to sell your crops for SFL _[need to change the shop image]_
3, Buy more seeds using your SFL
4, Plant seeds and wait

How to Gather

1, Visit the Blacksmith
2, Purchase tools
3, Use tools on resource nodes
4, View resources in your inventory

How to Expand

1, Click on expand area
2, Provide required Resources
3, Sign on chain transaction
3, Wait for timer to complete

How to gain XP

1, Click on the build hammer
2, Install a firepit
3, Click on the firepit and cook food
4, Click on your bumpkin
5, Give food to your Bumpkin

How to save

1, Click on the boat for a temporary save
2, Click on the settings cog
3, Click Sync on Chain
4, Sign On Chain Transaction - Game is now saved on chain.

This is a basic guide to get you started, if you want to learn more about the game and dive deeper into the documentation you can visit our website

[button] back to game
[button] visit docs
(two buttons 50% each)
